src/app/collections/collection.directive.js

```diff
@@ -31,9 +31,10 @@
     }
 
     /** @ngInject */
-    function CollectionController ($filter, $localStorage, $log, $timeout, RELOAD_TIMEOUT, collectionsService, networkService) {
+    function CollectionController ($filter, $localStorage, $log, $timeout, API, authService, collectionsService, networkService, RELOAD_TIMEOUT) {
```

Comment:
Linting should give an error about non-alphabetized parameters on this line

Reply:
Im confused about this when i saw it in the linter output...isnt this is alpha order?

Reply:
Capitals come before lower case in the rule-set